### PR TITLE
docs(astro-sst): update wrong astro output

### PR DESCRIPTION
**Pull request by unrevised6419**
The only valid astro output values are 'static' | 'server' | 'hybrid'

image

### CODE REVIEW

**Comment by jbrown** [comment]
I had the same issue after following the Astro guide. Check this commit from 2 weeks ago:

https://github.com/sst/ion/commit/4f55966b36f4854f5a256d8c46d2e3e051dab919

@jayair is there an explanation for this change?

The [README for astro-sst](https://github.com/sst/sst/tree/master/packages/astro-sst) still says to put `output: "server"`

**Comment by jayair** [comment]
Oh man somebody opened an issue about that change and I put it in. Now I can't find it.

@iamandrewluca can you change this back to "server" then.

File: www/src/content/docs/docs/start/aws/astro.mdx [hunk 1]
```diff
@@ -43,7 +43,7 @@ It'll also ask you to update your `astro.config.mjs` with something like this.
 + import aws from "astro-sst";
 
 export default defineConfig({
-+  output: "dist",
++  output: "hybrid",
```
Comment (unrevised6419) [inline www/src/content/docs/docs/start/aws/astro.mdx]:
```suggestion
+  output: "server",
```

**Comment by jayair** [comment]
Actually, let me check with @fwang because this what we do internally:

https://github.com/sst/ion/blob/dev/www/astro.config.mjs#L177

**Comment by unrevised6419** [comment]
@jayair too late, changed to `server` already ))

**Comment by unrevised6419** [comment]
Fixed in https://github.com/sst/ion/commit/6e3b26b03a4e9d9432de0ccdc622fb8e81551b95

**Pull request closed without merging**